cmd/burrow/commands/keys.go

```diff
@@ -46,9 +48,26 @@ func Keys(output Output) func(cmd *cli.Cmd) {
 
 		cmd.Command("server", "run keys server", func(cmd *cli.Cmd) {
 			keysDir := cmd.StringOpt("dir", keys.DefaultKeysDir, "specify the location of the directory containing key files")
+			badPerm := cmd.BoolOpt("allow", false, "Allow unix key file permissions to be readable other than user")
```

Comment:
feel like this should be a bit longer. `allow-bad-perm` or `ignore-perms` or something like that